src/Settings/settings.json

```diff
@@ -1182,5 +1182,25 @@
     "type": "bool",
     "default": false,
     "notr": true
+  },
+  {
+    "name": "WakaTime/Path",
```

Comment:
Let it depend on "WakaTime/Enable". So if WakaTime is enabled then only path and API keys settings are enabled.